python/ribasim_api/README.md

```diff
@@ -0,0 +1,23 @@
+# Ribasim API
+
+`ribasim_api` is an extension to [xmipy](https://pypi.org/project/xmipy/) for the Ribasim API.
+
+The `ribasim_api` can be used to access functionality in the eXtended Model Interface (XMI) wrapper (XmiWrapper)
+and additional functionality specific to the Ribasim API.
+
+
+`ribasim_api` can be installed by running
+
+```
+pip install ribasim-api
+```
+
+or
+
+```
+conda install -c conda-forge ribasim-api
+```
```

Comment:
Are you picking up #287?

Reply:
Already published it on PyPI and invited you :)

Reply:
Accepted :)